python/lumifit/reconstruction.py

```diff
@@ -122,12 +122,12 @@ def create_reconstruction_job(
             print("error: thought dir does not exists but it does...")
 
     # * Those are already in the experiment config, no need to write them again
-    # write_params_to_file(
-    #     attr.asdict(reco_params), pathname_full, "reco_params.config"
-    # )
-    # write_params_to_file(
-    #     attr.asdict(align_params), pathname_full, "align_params.config"
-    # )
+    write_params_to_file(
+        attr.asdict(reco_params), pathname_full, "reco_params.config"
+    )
+    write_params_to_file(
+        attr.asdict(align_params), pathname_full, "align_params.config"
+    )
```

Comment:
Because we need to write new `reco_params.config` and `align_params.config` in folders such as`dpm_elastic_theta_2.7-13.0mrad_recoil_corrected/ip_offset_XYZDXDYDZ_0.0_0.0_0.0_0.0_0.0_0.0/beam_grad_XYDXDY_0.0_0.0_0.0_0.0/no_geo_misalignment/100000/1-100_xy_m_cut_real/no_alignment_correction/`.

Reply:
As agreed, we should keep them, if only to document which reco parameters have been used.